deepmd/dpmodel/model/base_model.py

```diff
@@ -35,7 +35,7 @@ class BaseBaseModel(ABC, PluginVariant, make_plugin_registry("model")):
 
         def __new__(cls, *args, **kwargs):
             if inspect.isabstract(cls):
-                cls = cls.get_class_by_type(kwargs.get("type", "standard"))
+                cls = cls.get_class_by_type(kwargs.get("type", "ener"))
```

Comment:
should get the model type from the fitting type instead of `ener`

Reply:
Or modify `get_class_by_type`

Reply:
in case there is no fitting type, which Model do we use?